py/picca/io.py

```diff
@@ -232,16 +238,49 @@ def read_blinding(in_dir):
                                                            + '/*.fits.gz')
     filename = files[0]
     hdul = fitsio.FITS(filename)
-    if "LAMBDA" in hdul: # This is for ImageHDU format
+    # This is for ImageHDU format
+    if "LAMBDA" in hdul:
         header = hdul["METADATA"].read_header()
+
+        # read blinding
         blinding = header["BLINDING"]
-    else: # This is for BinTable format
+
+        # check if we are doing metal forests
+        if "MIN_RF_WAVE" in header and "MIN_RF_WAVE" in header:
+            lya_wave = ABSORBER_IGM["LYA"]
+            min_rf_wave = header["MIN_RF_WAVE"]
+            max_rf_wave = header["MAX_RF_WAVE"]
+            if min_rf_wave > lya_wave and max_rf_wave > lya_wave:
+                if lambda_abs2 is None:
+                    lambda_abs2 = lambda_abs
+                blinding_tracers = ["LYA", "QSO"]
+                if tracer1 not in blinding_tracers and tracer2 not in blinding_tracers:
+                    # since we are doing metal forests, overwrite the read blinding
+                    # and force data to not being blind
+                    blinding = "none"
+
+    # This is for BinTable format
+    else:
         header = hdul[1].read_header()
         if "BLINDING" in header:
             blinding = header["BLINDING"]
         else:
             blinding = "none"
 
+        # check if we are doing metal forests
+        if "MIN_RF_WAVE" in header and "MIN_RF_WAVE" in header:
+            lya_wave = ABSORBER_IGM["LYA"]
+            min_rf_wave = header["MIN_RF_WAVE"]
+            max_rf_wave = header["MAX_RF_WAVE"]
+            if min_rf_wave > lya_wave and max_rf_wave > lya_wave:
+                if lambda_abs2 is None:
+                    lambda_abs2 = lambda_abs
+                blinding_tracers = ["LYA", "QSO"]
+                if tracer1 not in blinding_tracers and tracer2 not in blinding_tracers:
```

Comment:
@iprafols - I think you want to have an "or" here, since we don't want to blind QSO x CIV either. Basically, you want to blind only when both tracers are either LyA or QSO

Reply:
@iprafols - it looks like you have fixed this above (for the ImageHDU format) but you have not fixed it for the BinTable format. It still says "and" instead of "or"